ts/webui/src/components/common/ExpandableDetails/TrialLog.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+const TrialLog = (props): any => {
+    const { logStr, logName } = props;
+    const isHyperlink = /^http/gi.test(logStr);
```

Comment:
I would prefer `logStr.toLowerCase().startsWith('http')`. I think regex is hard to read.

Reply:
ok